src/Ninja/G1CP/Content/Tests/test015.d

```diff
@@ -9,22 +9,17 @@ func int G1CP_Test_015() {
     // Define possibly missing symbols locally
     const int ATR_STRENGTH = 4;
 
-    // Check status of the test
-    var int passed; passed = TRUE;
-
-    // Check if the dialog function exists
-    var int funcId; funcId = MEM_GetSymbolIndex("DIA_Horatio_HelpSTR_LEARN_NOW");
-    if (funcId == -1) {
-        G1CP_TestsuiteErrorDetail("Dialog function 'DIA_Horatio_HelpSTR_LEARN_NOW' not found");
-        return FALSE;
-    };
+    // Prior checks
+    var int funcId; funcId = G1CP_Testsuite_GetDialogFuncId("DIA_Horatio_HelpSTR_LEARN_NOW");
```

Comment:
I think we won't need the `funcId` here actually. We only need to confirm the function's existence. Below we can use `MEM_CallByString("DIA_Horatio_HelpSTR_LEARN_NOW")` instead of `MEM_CallById(funcId)`. So `G1CP_Testsuite_CheckDialogFunc` would suffice. I think that will aid in cleaning up the tests and make them more readable.

Reply:
That may go for all function getter-functions.